bases/br_cgu_servidores_executivo_federal/servidores_militares_cadastro/table_config.yaml

```
@@ -0,0 +1,355 @@
+table_id: servidores_militares_cadastro # AUTO GENERATED
+dataset_id: br_cgu_servidores_executivo_federal # AUTO GENERATED
+
+last_updated: 2020-10-29 # AUTO GENERATED
+
+# Descreva a tabela. Essas são as primeiras frases que um usuário vai ver.
+# Você não precisa ser muito conciso. Sinta-se a vontade para dar exemplos de
+# como usar os dados.
+# Se souber, liste também aplicações: pesquisa, apps, etc. que usem os dados.
+description: |  # REQUIRED
+    Registro (cadastro) dos servidores públicos militares do poder executivo federal, tanto da administração direta quanto indireta. Também inclui ocupantes de cargos comissionados. Não inclui registro de militares reformados (aposentados), terceirizados, empregados de empresas públicas, servidores civis e servidores de outros poderes e esferas.
+
+    Os registros se repetem em cada mês (um servidor ativo de fev/2015 a nov/2018 terá um registro para cada mês, com os dados referentes àquele mês (cargo ocupado, etc).
+
+# Qual organização/departamento/pessoa tratou os dados?
+# As vezes há um ponto intermediário entre os dados originais e subir na Base dos Dados.
+# Se essa pessoa é você, preencha abaixo com suas informações.
+treated_by:
+    name: Henrique Xavier  # REQUIRED
+    code_url: https://github.com/basedosdados/mais/tree/master/bases/br_cgu_servidores_executivo_federal/code # REQUIRED
+    website: <onde encontrar os dados tratados>
+    email: hsxavier@gmail.com
+
+# Quem está completando esse arquivo config?
+published_by:
+    name: Henrique Xavier  # REQUIRED
+    code_url: https://github.com/basedosdados/mais/tree/master/bases/br_cgu_servidores_executivo_federal/code # REQUIRED
+    website: http://henriquexavier.net/
+    email: hsxavier@gmail.com
+
+# Se houve passos de tratamento, limpeza e manipulação de dados, descreva-os aqui.
+treatment_description: | 
+    A descrever no futuro.
+
+# Com qual frequência a base é atualizada?
+# Opções: hora | dia | semana | mês | 1 ano | 2 anos | 5 anos | 10 anos | único | recorrente
+data_update_frequency: mês # REQUIRED 
+
+# Nível da observação: quais colunas identificam uma linha unicamente?
+# Escolha todas as opções necessárias.
+# Regras: minúsculo, sem acento, singular.
+# Opções (exemplos): pais, estado, municipio, cidade, hora, dia, semana, mes, ano, etc.
+observation_level:
+    - id_servidor
```

o que torna cada linha única aqui, também não seria por `id_atividade`?
```suggestion
    - id_servidor
    - id_atividade
```